<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BurstStatisticsView</class>
 <widget class="QWidget" name="BurstStatisticsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,1,0,1">
        <item row="1" column="0" colspan="4">
         <widget class="QFPlotter" name="widPlotSize" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="4">
         <widget class="QFPlotter" name="widPlotRate" native="true"/>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QFPlotter" name="widPlotDuration" native="true"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Burst Rate Distribution</string>
          </property>
          <property name="buddy">
           <cstring>widPlotRate</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Max Burst Size</string>
          </property>
          <property name="buddy">
           <cstring>spinBoxMaxBurstSize</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBoxMaxBurstSize">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Burst Duration Distribution</string>
          </property>
          <property name="buddy">
           <cstring>widPlotDuration</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Burst Size Distribution</string>
          </property>
          <property name="buddy">
           <cstring>widPlotSize</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,1,0,1">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>t_g-t_r</string>
          </property>
          <property name="buddy">
           <cstring>widPlotDuration</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QFPlotter" name="widPlotTg" native="true"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Burst Rate Distribution</string>
          </property>
          <property name="buddy">
           <cstring>widPlotRate</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QFPlotter" name="widPlot" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,0">
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxBins">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>177</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Source</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxSource</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Bins</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxBins</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="6">
        <widget class="QTableView" name="tableInfo"/>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="pushButtonRefresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="6">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Parameter</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="comboBoxSource">
         <item>
          <property name="text">
           <string>all bursts</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>selected bursts</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFPlotter</class>
   <extends>QWidget</extends>
   <header>qfplotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonRefresh</tabstop>
  <tabstop>spinBoxMaxBurstSize</tabstop>
  <tabstop>spinBoxBins</tabstop>
  <tabstop>tableInfo</tabstop>
  <tabstop>comboBoxSource</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
